packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -565,6 +565,114 @@ describe("calcite-date-picker", () => {
     await page.waitForChanges();
     expect(await datePicker.getProperty("value")).toEqual(["2020-09-15", "2020-09-30"]);
   });
+
+  it("sets value to the selected day in the 2000s", async () => {
```

Comment:
Can you use `describe` blocks to group these tests? Maybe something like this:

```ts
describe("cross-century date values", () => {
  describe("PST", () => { /* ... */})
  describe("CEST", () => { /* ... */})
});
```
?

Applies to `input-date-picker` tests.